_extensions/quarto-journals/plos/filter.lua

```
@@ -0,0 +1,81 @@
+
+-- these classes, when placed on a span will be replaced
+ -- with an identical LaTeX command for PDF output
+ local texMappings = {
+  "nameref"
+}
+
+return {
+  {
+    Span = function(el)
+      local contentStr = pandoc.utils.stringify(el.content)
+        for i, mapping in ipairs(texMappings) do
```

Luacheck (1.2.0)

[warning] unused loop variable 'i'

(W213)